dogfood/main.tf

```diff
@@ -266,8 +266,12 @@ resource "coder_agent" "dev" {
     sudo service docker start
     # Install playwright dependencies
     # We want to use the playwright version from site/package.json
+    # Check if the directory exists At workspace creation as the coder_script runs in parallel so clone might not exist yet.
+    while ! [[ -f "${local.repo_dir}/site/package.json" ]]; do
```

Comment:
I just updated my workspace and this is looping infinitely, maybe the "~" is getting interpreted literally due to the quotes.

```
coder@dev27:~/coder$ [[ -f "~/coder/site/package.json" ]] && echo true || echo false
false
coder@dev27:~/coder$ [[ -f ~/coder/site/package.json ]] && echo true || echo false
true
```

I think maybe we will have to omit the quotes, expand the ~ beforehand, or make it relative.  Thoughts?

Reply:
Or is it possible mine is just messed up somehow?  Maybe there is a parameter I have to change.

Reply:
Aha `Coder Repository Base Directory` was set to `~` and I changed it to `/home/coder`.  Feel free to ignore me, although maybe we want to change this anyway just in case others run into the same problem.

Reply:
Yes I agree feel free to push a fix